### DISCUSSION TITLE
XDM Integration Module, Unable to connect with XDM

### DISCUSSION THREAD

**Discussion by camerojosue1** [subhra74/xdm#986] [General]
I have this problem, I uninstalled the application and reinstalled it and nothing happened, this is version 8.0.18-beta, I currently have the linux mint operating system

![Captura de pantalla de 2023-01-30 11-31-13](https://user-images.githubusercontent.com/123970692/215521413-7a65da29-3af5-4ec0-84a5-600969314c98.png)

**Comment by biladina** [comment]
is the XDM running in the backgroud??

**Reply by realyukii** [reply]
same issue :(
I have make sure that the XDM running on background

**Comment by functionalatbest** [comment]
I am having the same issue on Brave with Windows. It worked for literally one video and then seconds later stopped working and no amount of restarting, reinstalling, and refreshing seems to be fixing it

**Comment by cbag1** [comment]
Disable and enable again. That should works now

**Reply by nb00100** [reply]
👍
